client/src/pages/OpeningDashboardPage/CreateOpeningPage.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react";
+import Sidebar from "../../components/Sidebar";
+import { createOpenings } from "./../../api/opening";
+import { useSelector } from "react-redux";
+
+const CreateOpeningPage = () => {
+  const userId = useSelector((state) => state.auth.user._id) || null;
+
+  const [formData, setFormData] = useState({
+    company: "",
+    number: 1,
+  });
+  const [validMessage, setValidMessage] = useState("");
+
+  const handleChange = (e) => {
+    setFormData({
+      ...formData,
+      [e.target.id]: e.target.value,
+    });
+  };
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    if (formData.number < 1) {
+      setValidMessage("Number must be greater than 0");
+      return;
+    }
+
+    createOpenings(formData, userId);
+    toast.success("Form successfully submitted");
```

Comment:
shouldn't add this toast here, but add in createOpenings. What if createOpenings fail to create, the toast.success is still sequentially the next step